AddBeerViewController.h

```diff
@@ -0,0 +1,17 @@
+//
+//  AddBeerViewController.h
+//  WhatsOnTap
+//
+//  Created by Sarmila on 6/22/16.
+//  Copyright © 2016 Detroit Labs. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "EstablishmentDetailViewController.h"
+#import "Beer.h"
+
+@interface AddBeerViewController : UIViewController
+@property (weak, nonatomic) IBOutlet UITextField *addBeerText;
+@property (nonatomic) Beer *beerToBeAdded;
```

Comment:
I think it's better to err on the side of being explicit with strong and weak declarations. Just my opinion though